src/main/java/io/logz/jmx2logzio/clients/JolokiaClient.java

```diff
@@ -143,20 +146,20 @@ public void setExtraDimensions(List<Dimension> extraDimensions) {
      * @param response A response map from the Jolokia server
      * @return a list of logz.io metrics
      */
-    private List<Metric> getMetricsForResponse(Map<String, Object> response) {
+    private Metric getMetricsForResponse(Map<String, Object> response) {
```

Comment:
```suggestion
    private Metric getMetricForResponse(Map<String, Object> response) {
```

Reply:
I'm not sure if we should change the name.. It is a few metrics in a single doc..